.circleci/config.yml

```diff
@@ -313,7 +320,7 @@ jobs:
           path: /tmp/fake-mailer/
           destination: fake-mailer
   php7_integration:
-    executor: wordpress_mysql80_php73
```

Comment:
Since this executor is no longer used, please remove it also from the list of executors.

Reply:
Fixed.